govstack_api/building_blocks/bb_digital_registries/registries/insuree_registry.py

```diff
@@ -126,27 +38,28 @@ def get_record_field(self, mapped_data, field=None, extension=None):
     def retrieve_filtered_records(self, mapped_data):
         return self.get_record(mapped_data, only_first=False)
 
-    def update_registry_record(self, mapped_data_query, mapped_data_write):
+    def manage_registry_record(self, mutation_name, mapped_data_query, mapped_data_write=None):
```

Comment:
Can we move this function to base registry? What are points that would be unique for particular registries? We could work together to make it generic and base on config.

Reply:
Unique could be required fields but in my opinion it depends on the more complicated registries with relations etc. For now we can rename one variable and it will be reusable since there are default values set via config and only uuid is added as must have.